mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -139,7 +148,12 @@ The following arguments are supported:
 * `auto_healing_policies` - (Optional) The autohealing policies for this managed instance
 group. You can specify only one value. Structure is [documented below](#nested_auto_healing_policies). For more information, see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/creating-groups-of-managed-instances#monitoring_groups).
 
-* `stateful_disk` - (Optional)) Disks created on the instances that will be preserved on instance delete, update, etc. Structure is [documented below](#nested_stateful_disk). For more information see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/configuring-stateful-disks-in-migs).
+* `all_instances_config` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
+  Properties to set on all instances in the group. After setting
+  allInstancesConfig on the group, you must update the group's instances to
+  apply the configuration.
+
+* `stateful_disk` - (Optional) Disks created on the instances that will be preserved on instance delete, update, etc. Structure is [documented below](#nested_stateful_disk). For more information see the [official documentation](https://cloud.google.com/compute/docs/instance-groups/configuring-stateful-disks-in-migs).
```

Comment:
missing stateful disk nested subfield explanation

Reply:
stateful_disk documentation and #nested_stateful_disk was already in the documentation. I just added fragment about all_instances_config.  I think we should look at missing/inconsistent parts of the documentation in the separate issues/PR. WDYT?

Reply:
ahh I see, thanks for the notice !